hack/build/run-lint-checks.sh

```diff
@@ -27,4 +27,27 @@ if [[ ${out} ]]; then
     echo "${out}"
     ec=1
 fi
+# Once we get everything fixed up we'll just have a single lint check
+# in the meantime add pkgs as they're ready
+# controller
+out="$(golint pkg/controller/...)"
```

Comment:
what about a loop here to invoke `golint` and the various dirs?

Reply:
Sure, but this is just temporary, eventually we just run against all the src dirs in one shot.

Reply:
@j-griffith have you seen the allPkgs()s function in config.sh?  This will return all go files in the projects (sans `vendor/` and the bugged generated file`).